core/templates/dev/head/editor/feedback_tab/ThreadDataService.js

```diff
@@ -50,7 +50,8 @@ oppia.factory('threadDataService', [
   };
 
   var _fetchMessages = function(threadId) {
-    $http.get(_THREAD_HANDLER_PREFIX + threadId).success(function(data) {
+    $http.get(_THREAD_HANDLER_PREFIX 
+        + threadId.split(".")[1]).success(function(data) {
```

Comment:
I've made the backend changes. I looked at the frontend as well, and to me it doesn't seem like there's anymore changes needed in the frontend.